Arrowgene.Ddon.Test/Database/CharacterDatabaseTest.cs

```diff
@@ -18,7 +18,7 @@ public void TestCharacterOperations()
         {
             DatabaseSetting setting = new DatabaseSetting();
             setting.WipeOnStartup = true;
-            IDatabase database = DdonDatabaseBuilder.BuildSqLite(setting.SqLiteFolder, "character.test", true);
+            IDatabase database = DdonDatabaseBuilder.BuildSqLite(setting.DatabaseFolder, true);
```

Comment:
Would this be a problem? Unless DatabaseFolder is set to a different value wouldn't this overwrite the regular database when tests are run?

Reply:
The default DatabaseSetting constructor creates a DatabaseFolder like as `DatabaseFolder = Path.Combine(Util.ExecutingDirectory(), "Files/Database");`. This means that any normal test case execution should be relative to where the test is executing from.
Tests show that whether it's via the IDE or via `dotnet test`, it uses this folder: Arrowgene.DragonsDogmaOnline/Arrowgene.Ddon.Test/bin/Debug/net6.0/Files/Database